### PR TITLE
Fixing IDENTITY-5174 : Locales needs to be sorted

### PR DESCRIPTION
**Pull request by pulasthi7**

### CODE REVIEW

File: components/email-mgt/org.wso2.carbon.email.mgt.ui/src/main/resources/web/email-mgt/email-template-add.jsp [hunk 1]
```diff
@@ -37,6 +39,14 @@
     String emailTemplateType = request.getParameter("templateType");
     String username = request.getParameter("username");
     Locale[] availableLocale = Locale.getAvailableLocales();
+    Arrays.sort(availableLocale, new Comparator<Locale>() {
+
+        @Override
+        public int compare(Locale o1, Locale o2) {
+
```
Comment (hasinthaindrajee) [inline components/email-mgt/org.wso2.carbon.email.mgt.ui/src/main/resources/web/email-mgt/email-template-add.jsp]: Can you please remove unnecessary new lines ?

Comment (mefarazath) [inline components/email-mgt/org.wso2.carbon.email.mgt.ui/src/main/resources/web/email-mgt/email-template-add.jsp]: Shall we move this to the Util class?

File: components/email-mgt/org.wso2.carbon.email.mgt.ui/src/main/resources/web/email-mgt/email-template-add.jsp [hunk 2]
```diff
@@ -37,6 +39,14 @@
     String emailTemplateType = request.getParameter("templateType");
     String username = request.getParameter("username");
     Locale[] availableLocale = Locale.getAvailableLocales();
```
Comment (mefarazath) [inline components/email-mgt/org.wso2.carbon.email.mgt.ui/src/main/resources/web/email-mgt/email-template-add.jsp]: availableLocale --> availableLocales ?

Comment (pulasthi7) [inline components/email-mgt/org.wso2.carbon.email.mgt.ui/src/main/resources/web/email-mgt/email-template-add.jsp]: Fixed

**Pull request merged**